### PR TITLE
yaml config and minor updates

### PR DESCRIPTION
**Pull request by jemten**

### CODE REVIEW

File: mip_install.pl [hunk 1]
```diff
@@ -312,7 +218,7 @@
 
 ## Remove selected programs from installation
 if ( $parameter{skip_program} ) {
-  SKIP: 
+  SKIP:
     foreach my $skip ( @{ $parameter{skip_program} } ) {
```
Comment (henrikstranneheim) [inline mip_install.pl]: $skip -> $program

File: lib/MIP/Recipes/Install/Rhocall.pm [hunk 2]
```diff
@@ -10,7 +10,7 @@ use Carp;
 use English qw{ -no_match_vars };
 use Params::Check qw{ check allow last_error };
 use Cwd;
-use File::Spec::Functions qw{ catdir catfile };
+use File::Spec::Functions qw{ catdir catfile splitdir };
```
[the remark this answers is not in the document]
Comment (jemten) [reply lib/MIP/Recipes/Install/Rhocall.pm]: ??

[the remark this answers is not in the document]
Comment (henrikstranneheim) [reply lib/MIP/Recipes/Install/Rhocall.pm]: Ignore that comment

File: lib/MIP/Recipes/Install/Conda.pm [hunk 3]
```diff
@@ -852,4 +825,66 @@ sub _create_target_link_paths {
...
+            $log->fatal( q{Could not find conda installation path.}
+                  . $NEWLINE
+                  . q{Specify the path to conda using the --conda_dir_path flag}
+            );
+            exit 1;
+        }
+        ## Get the path to the conda base directory
+        my @conda_bin_in_paths = splitdir($conda_bin_in_path);
+
+        # Remove the tw last elements of the array (bin and conda)
```
Comment (henrikstranneheim) [inline lib/MIP/Recipes/Install/Conda.pm]: Tw -> two

**Pull request merged**